### ISSUE TITLE
Responsibilities of the team

### ISSUE THREAD

**Issue by zeusdeux** [iojs/iojs-hi#3]
So I can think of the following responsibilities. Do add to this list.

- io.js website translation
- Translation of io.js posts (for e.g., on [medium](https://medium.com/@iojs))
- Translation of documentation as necessary (??)
- Translation of io.js material released for the community

**Comment by manV** [comment]
We should ask mikeal for clarification...

**Comment by zeusdeux** [comment]
@manV yeah, i left him a question at iojs/website#125 :)

**Comment by manV** [comment]
:thumbsup:

**Comment by nmn** [comment]
I think it's important to find out how many people are actually interested in hindi translations. As we all probably know, most of the indian population does not use the web in Hindi. Programmers are already a subset, and they all can read English.

If we are doing hindi, it's important to decide how pedantic we want to get. Most people often know the english word over the hindi word. (e.g. train over लोहपदगामिनी)

Would love to hear your thoughts on this.

Also, I hope everyone has a set-up ready to actually type Hindi.

**Comment by zeusdeux** [comment]
I agree that most of programmers can already read English and will (in all probability) read the posts put out in english. But that shouldn't stop content going out in hindi imho. The cost benefit analysis isn't in our favour right now but I see this as a long term investment.

Maybe we'll find an audience that we didn't know of. Maybe people will use the hindi version in conjunction with the english version to gain clarity on parts that they didn't understand in the english version. I don't know. I think it's an experiment that's worth my while though.

Also, I am thinking more hinglish than pure hindi tbh.

Oh and yeah, hope you guys have your hindi typing setups ready. :)

**Comment by manV** [comment]
Totally agree with you @zeusdeux, it might be a long term investment. +1 for hinglish against pure hindi.

**Comment by nmn** [comment]
Guys, I've already done a translation of the Medium post. Going forward, I will try to come and contribute translations whenever possible.

I would like it if one of you took the lead in publishing the content.

**Comment by manV** [comment]
I'll do it, but I can't decide where to post it....

**Comment by nmn** [comment]
Why not post it to medium.com as well.

Also do  you guys know of any place where hindi exists online? There used to be webduniya when I was in school like 10 years ago.

**Comment by manV** [comment]
The post is up on Medium

**Comment by zeusdeux** [comment]
I doubt there are any big programming communities that use Hindi as a language. We have to start seeding normal communities with Hindi posts so that at some point there can be a larger spin off them imo.

**Comment by nmn** [comment]
@zeusdeux I guess that makes sense.
